src/Hadolint/Formatter/Codeclimate.hs

```diff
@@ -83,14 +82,6 @@ checkToIssue RuleCheck {..} =
       impact = severityText (severity metadata)
     }
 
-severityText :: Severity -> String
```

Comment:
This code is required by `codeclimate` as those are the values that it can read

Reply:
Oops, how embarrassing. I mistook this to be the same as the other `severityText`